src/WebJobs.Script.WebHost/Security/Authentication/Jwt/ScriptJwtBearerExtensions.cs

```diff
@@ -118,5 +126,53 @@ public static TokenValidationParameters CreateTokenValidationParameters()
 
             return result;
         }
+
+        private static string IssuerValidator(string issuer, SecurityToken securityToken, TokenValidationParameters validationParameters)
+        {
+            if (!validationParameters.ValidIssuers.Any(p => string.Equals(issuer, p, StringComparison.OrdinalIgnoreCase)))
+            {
+                throw new SecurityTokenInvalidIssuerException("IDX10205: Issuer validation failed.")
```

Comment:
Using the same error code as the base validator [here](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/f5fef2269a6eea5ba52d707d7235acf6a504db9f/src/Microsoft.IdentityModel.Tokens/Validators.cs#L317C19-L317C19)